src/packages/@ncigdc/components/Project.js

```diff
@@ -110,7 +110,7 @@ const Project = (
           <DownloadButton
             disabled={!biospecimenCount}
             filename={`biospecimen.project-${projectId}`}
-            url={urlJoin(process.env.REACT_APP_API, 'cases')}
+            url={urlJoin(process.env.REACT_APP_GDC_AUTH, 'api/cases')}
```

Comment:
add a new ENV called `REACT_APP_GDC_AUTH_API` since 99% of the time it needs the /api part - the only time it doesn't is for the `/user` request